src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,24 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private String modelTruck;
```

Comment:
```suggestion
    private String model;
```
We already know that this field belongs to Truck)